client/src/browse/list/ItemCard.js

```diff
@@ -0,0 +1,49 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+
+import { BsCardImage } from "react-icons/bs";
+
+const ItemCard = (props) => {
+  const clickHandler = () => {
```

Comment:
@MarkAlperin just commenting about destructuring again. Would recommend the same here.